python/packages/autogen-ext/src/autogen_ext/models/_openai/_openai_client.py

```diff
@@ -553,7 +554,7 @@ async def create_stream(
         messages: Sequence[LLMMessage],
         tools: Sequence[Tool | ToolSchema] = [],
         json_output: Optional[bool] = None,
-        extra_create_args: Mapping[str, Any] = {},
+        extra_create_args: Mapping[str, Any] = {"stream_options": {"include_usage": True}},
```

Comment:
My gut feeling is that we should try to be consistent with the `openai` client's behavior and have user set this option rather than making it a default. There are users using local model with LiteLLM proxy that may not support this parameter. So, there will be bad surprises in those cases. I think we can instead document this behavior in the `__doc__` string of this method, as well as in the introduction to model clients. https://github.com/microsoft/autogen/blob/main/python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/model-clients.ipynb

Reply:
Thank you for your feedback and agree, I have refactored to default `openai` clients behavior, added a doc string and updated the example. As there are no other doc strings in this yet, i have attempted to replicate style from where i could find, please refine or guide appropriately.

Reply:
Thank you for the edits!